src/main/java/com/amazon/aws/iot/greengrass/configuration/common/Configuration.java

```diff
@@ -41,6 +41,10 @@ public class Configuration {
 
     private String configurationArn;
```

Comment:
this is what we're using today to determine the group, right? So why are we adding in 2 fields. Adding in 1 field should be sufficient for our needs. The `onBehalfOf` field.

Reply:
For conflict resolution we only need `root` group value. I would let you drive the field name. Let me know what you think. 

Also we have placed these values as iot Arns, where the configurationArn is a greengrass Arn.